graphql_api/tests/test_coverage_analytics_measurements.py

```diff
@@ -0,0 +1,122 @@
+import datetime
+from unittest.mock import patch
+
+from django.test import TransactionTestCase, override_settings
+from django.utils import timezone
+
+from codecov_auth.tests.factories import OwnerFactory
+from core.tests.factories import (
+    RepositoryFactory,
+)
+from timeseries.models import Interval
+
+from .helper import GraphQLTestHelper
+
+
```

Comment:
These tests are basically just copied over from `test_repository_measurements.py` plus small fixes to reflect the new nesting of the `coverageAnalytics` type.